kf4cic.c

```diff
@@ -211,6 +211,42 @@ static EFI_STATUS load_and_start_tos(void)
 
 	return ret;
 }
+
+static EFI_STATUS update_rollback_indexes()
+{
+	EFI_STATUS ret;
+	AvbOps *ops;
+	AvbSlotVerifyResult verify_result;
+	AvbSlotVerifyData *slot_data = NULL;
+	UINT8 boot_state = BOOT_STATE_GREEN;
+	const char *requested_partitions[] = {"tos",
+		NULL};
+
+	ops = avb_init();
+	if (!ops) {
+		error(L"Failed to init avb");
+		return EFI_OUT_OF_RESOURCES;
+	}
+
+	verify_result = avb_slot_verify(ops,
```

Comment:
MINOR: Even if some other header file might incude header where avb_slot_verity, get_avb_result & avb_update_stored_rollback_indexes_for_slot are introduced I would still include the original header file (android_vb2.h) in this file.